### ISSUE TITLE
Status of the adapter

### ISSUE THREAD

**Issue by lukealexmiller** [Dandandan/dbt-athena#7]
Thanks for the great work on this! 
Very keen to use dbt at our co and we're using Athena in production. 
I gave the adapter a try using `pip install dbt==1.14.0 && pip install git+https://github.com/Dandandan/dbt-athena.git`

However, running 'dbt debug' I get
`Profile loading failed for the following reason:
Runtime Error
  Credentials in profile "athena_data_warehouse", target "awscatalog" invalid: Runtime Error
    Could not find adapter type athena!`

The profile and a project yml files have the relevant database/schema settings and pass debug.

I realise the repo does say 'work-in-progress', should this be functional right now or should I check back in on progress later.

Thanks again!

**Comment by Dandandan** [comment]
Thank you for raising the isse.

It should be functional, although still a lot of rough edges.

Can you share the structure of your project /  (with secrets removed)?

**Comment by lukealexmiller** [comment]
Hey! 
dbt_project.yml is as follows:

```
name: 'fy_analytics'
version: '1.0.0'
profile: 'athena_data_warehouse'

source-paths: ["models"]
analysis-paths: ["analysis"]
test-paths: ["tests"]
data-paths: ["data"]
macro-paths: ["macros"]

target-path: "target" 
clean-targets: 
    - "target"
    - "dbt_modules"


models:
  my_new_project:
      example:
          materialized: view
```

and profiles.yml is

```
athena_data_warehouse:
  target: awscatalog
  outputs:
    awscatalog:
      type: athena
      database: fy_reporting
      schema: athena_data_warehouse
      s3_staging_dir: athena-data-warehouse
      region_name: eu-west-1
      threads: 8
```

Having removed the id and secret key spec.

**Comment by Dandandan** [comment]
I don't see anything suspicious yet.
One thing I notice is that database probably should be awscatalog (AWS refers to the catalog as "awscatalog") and the schema is referred in AWS as a "database". Not really intuitive, but that's how it currently maps. I am not sure why the athena type is not found...

**Comment by Dandandan** [comment]
Can you try installing it again?

There was a small error in the dependency listed in setup.py (PyAthenaJDBC instead of PyAthena).

Otherwise, could you post the output of logs/dbt.log ?

**Comment by Dandandan** [comment]
@lukealexmiller

**Comment by lukealexmiller** [comment]
Thanks, that seems to have fixed the problem and I'm now able to connect. 👍 

Additional Q: Is it expected that the example models which come with `dbt init` can be executed against Athena using `dbt run`? I don't see why at least [my-first-dbt-model](https://github.com/fishtown-analytics/dbt-starter-project/blob/master/models/example/my_first_dbt_model.sql) should not be run successfully, but I have an error trying to access relation.database method because relation is of NoneType.

**Comment by Dandandan** [comment]
Thanks for checking. I checked running

https://github.com/fishtown-analytics/dbt-starter-project/blob/master/models/example/my_first_dbt_model.sql#L16

in my setup, and that model succeeds here, so there should be some difference between your setup and mine. Let's see whether we can find this out.

**Comment by Dandandan** [comment]
Which version of dbt are you running?
As I see 0.15.1 doesn't contain drop at line line 355.

https://github.com/fishtown-analytics/dbt/blob/dev/0.15.1/core/dbt/adapters/cache.py

Could you run pip install dbt==0.15.1 just to be sure?

**Comment by lukealexmiller** [comment]
Ah, you are correct. I still had dbt==0.14.0 installed, but notice that you've just merged support for 0.15.1. The examples execute correctly now.

Thanks for your help, and great work!

**Comment by Dandandan** [comment]
Great, thanks for testing! Please let me know when you encounter other issues!
